pages/02.general/11.https-filtering/01.https-filtering-known-issues/docs.en.md

```diff
@@ -6,66 +6,66 @@ taxonomy:
+AdGuard’s behavior depends on the platform. AdGuard for Windows uses Microsoft CryptoAPI to perform OCSP checks, and it has all the drawbacks that Chrome developers describe. AdGuard for Android doesn’t perform OCSP checks whatsoever.
 
 **Our plans:**
 
 * Add a possibility to use CRLSets (an alternative to the OCSP by Google) to the validation library
 * Add the OCSP Must-Staple support to the validation library
 * Stop using the CryptoAPI in favor of cross-platform implementation of the OCSP revocation checking
 
-**ETA:** The first certificate validation library release is planned for July-August 2017.
+**ETA:** This will be implemented after we complete the development of the new cross-platform filtering engine, which is planned for December 2017 — January 2018.
 
 #### HPKP support
 
 [HPKP (HTTP Public Key Pinning)](https://en.wikipedia.org/wiki/HTTP_Public_Key_Pinning) is a mechanism that allows websites to defend against the unauthorized attempts to intercept the encrypted (HTTPS) connections to them. This standard is not too widespread so far (of the Alexa top1000 websites, less than 1% uses HPKP) due to the high risks of using and sophisticated configuration process. Some experts even go as far as saying that HPKP is [de-facto dead](https://blog.qualys.com/ssllabs/2016/09/06/is-http-public-key-pinning-dead). Nonetheless, it remains the only protection mechanism to keep working should it happen the center of certification becomes compromised.
 
-There’s one serious problem with using HTTPS filtering - browsers completely ignore HPKP (because it isn’t used alongside with a local root certificate). This is a common issue for all programs that filter HTTPS. The only possible solution is to implement the HPKP standard within Adguard.
+There’s one serious problem with using HTTPS filtering - browsers completely ignore HPKP (because it isn’t used alongside with a local root certificate). This is a common issue for all programs that filter HTTPS. The only possible solution is to implement the HPKP standard within AdGuard.
 
 **Our plans:**
 
 * Add the HPKP support to the certificate validation library
-* Change the Adguard’s behavior for the cases when an invalid certificate is spotted. Instead of the current solution (ceasing the filtering for this domain) we should warn the user about the problem.
+* Change the AdGuard’s behavior for the cases when an invalid certificate is spotted. Instead of the current solution (ceasing the filtering for this domain) we should warn the user about the problem.
 
-**ETA:** The first certificate validation library release is planned for July-August 2017.
+**ETA:** Just as with OCSP revocation support, we wait until the new filtering engine development is finished. ETA is December 2017 — January 2018.
```

Comment:
Google's "intent to remove" HPKP support may change our plans:
https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/he9tr7p3rZ8